src/components/ListItem.jsx

```diff
@@ -44,6 +47,18 @@ export function ListItem({ listToken, item, itemId }) {
 		}
 	}, [is24HoursPassed, itemId, listToken, checked]);
 
+	const handleDelete = async () => {
+		try {
+			await deleteItem(listToken, itemId);
+		} catch (error) {
```

Comment:
one non-critical suggestion: since the deleteItem function already has a try-and-catch error clause, do you still need this try-and-catch error handling here?

Reply:
@shangguanwang You have raised a good point, so we have removed the try-and-catch error handling. Thank you so much! ❤️